addons/product/models/product_template.py

```diff
@@ -538,7 +538,18 @@ def copy(self, default=None):
             default = {}
         if 'name' not in default:
             default['name'] = _("%s (copy)", self.name)
-        return super(ProductTemplate, self).copy(default=default)
+
+        res = super(ProductTemplate, self).copy(default=default)
+
+        # Since we don't copy the attribute lines directly, we need to match the extra price.
```

Comment:
```suggestion
        # Since we don't copy the product template attribute values, we need to match the extra
        # price.
```
The comment is kinda wrong, since we do copy the attribute lines.